bootstrap.sh

```diff
@@ -29,6 +29,14 @@ install_manifest() {  # $1 = name of the manifest file
     curl --location --remote-name https://raw.githubusercontent.com/$MANIFESTS_SOURCE_REPOSITORY/$MANIFESTS_SOURCE_FOLDER/$1.pp --output $BOOTSTRAP_MANIFESTS_DESTINATION_FOLDER
 }
 
+function run_puppet {
```

Comment:
A link to the Puppet issue that explains why we have to wrap the execution would be welcome.